router/batchrouter/batchrouter.go

```diff
@@ -2015,6 +2016,7 @@ func loadConfig() {
 	config.RegisterBoolConfigVariable(false, &disableEgress, false, "disableEgress")
 	config.RegisterBoolConfigVariable(true, &readPerDestination, false, "BatchRouter.readPerDestination")
 	config.RegisterStringConfigVariable("", &toAbortDestinationIDs, true, "BatchRouter.toAbortDestinationIDs")
+	config.RegisterDurationConfigVariable(10, &netClientTimeout, false, time.Second, "BatchRouter.httpTimeout")
```

Comment:
Do we know the 95p time this requests needs ? Maybe 30 sec is safer.

Reply:
We don't have any stats around here I guess... Would it make sense to add one here? Also since this is a call within the same namespace, I thought we could have it at 10 seconds